src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -326,6 +326,33 @@ def db_create_replica(
         kwargs)
 
 
+# Creates a database from a database point in time backup or deleted database backup.
```

Comment:
Is this comment correct??  It seems like copy-pasta.

Reply:
Yep that's some tasty pasta